atm.js

```diff
@@ -1,21 +1,101 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+$(document).ready(function(){
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+  checkingAmount = $(".balance.checking");
+  savingAmount = $(".balance.saving");
 
-}
+  checkingInput = $("input.checkingInput");
+  savingInput = $("input.savingInput");
 
-function checkingWithdrawal(amount){
+  checkingDeposit = $(".checkingDeposit");
+  checkingWithdraw = $(".checkingWithdraw");
+  savingDeposit = $(".savingDeposit");
+  savingWithdraw = $(".savingWithdraw");
 
-}
+  checkingDeposit.on("click", depositCheckingMoney);
+  checkingWithdraw.on("click", withdrawCheckingMoney);
 
-function savingsDeposit(amount){
+  savingDeposit.on("click", depositSavingMoney);
+  savingWithdraw.on("click", withdrawSavingMoney);
```

Comment:
Can you think of a way to use one listener for each of these click handlers? i.e. maybe you can check the text of the button that's clicked on to decide what to do.